cw_core/lib/unspent_comparable_mixin.dart

```diff
@@ -0,0 +1,40 @@
+mixin UnspentComparable {
+  String get address;
+
+  String get hash;
+
+  int get value;
+
+  int get vout;
+
+  String? get keyImage;
+
+
+  bool isFrozen = false;
+
+  bool isSending = true;
+
+  String note = '';
+
+  void updateAdjustableFieldsFrom(UnspentComparable other) {
+    isFrozen = other.isFrozen;
+    isSending = other.isSending;
+    note = other.note;
+  }
```

Comment:
like this, would it modify the variables here or in the main object?
I don't think this approach is ideal

Reply:
reverted